android/app/src/fdroid/play/listings/ar/release-notes.txt

```diff
@@ -1,5 +1,9 @@
-- Edit Tracks: Change name, color, list and description of tracks, or delete them.
-- Open in Another App: Select any map point in Organic Maps and open in apps like taxi, delivery, transport, or other map apps.
-- Map Data: Updated as of October 1, now with added isolines for Egypt and improved for Adygea, Dagestan.
-- KML/KMZ/GPX: Fixed issues with importing files from different apps.
-- Right-to-left languages: Enhanced the place page, search, and about screens.
+- New OpenStreetMap data as of November 7.
```

Comment:
- Dots at the end of sentences are unnecessary, they take important limited character count.
- Using dots `• ` instead of dashes `- ` looks and reads better, they were always used previously.